src/components/DataTable.vue

```diff
@@ -24,36 +29,32 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, computed } from "@vue/composition-api";
+import { defineComponent, computed, reactive } from "vue";
 export default defineComponent({
   name: "data-table",
   props: {
     values: {
-      type: Array as any,
+      type: Object,
```

Comment:
```suggestion
      type: Array as PropType<any[]>,
```

Always give proper objects/arrays typescript types. Since this is a generic component, `any` on the generic slot of `Array<T>` is understandable.

Reply:
Applied changes